raml/schemas/definitions/session.json

```diff
@@ -0,0 +1,55 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "definitions":{
+        "project":      {"type": "string"},
+        "uid":          {"type": "string"},
+        "timestamp":    {"type": ["string", "null"], "format": "date-time"},
+        "timezone":     {"type": "string"},
```

Comment:
Just a nitpick, we could define uid, timestamp and timezone (in sessions and acquisitions) in the main container definitions (since we don't inherit via allOf).